### PR TITLE
Add flutter standards documentation

### PR DESCRIPTION
**Pull request by anand-OSM**
- Add dart coding style guideline
- Update Standard enforcing tools
- Add flutter lint and dart formatter tool setup

### CODE REVIEW

File: coding-standards/flutter.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
 # Flutter Coding Standards
 
-## Coding Style Guide
-
+## Flutter Coding Style Guidelines
+We adapted standards to design and program Flutter, from high-level architectural principles all the way to indentation rules, from Flutter offical git repo :</br>
 https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
+## Dart Coding Style Guidelines
+Dart Style Guide focuses primarily on Dart-specific conventions: https://dart.dev/effective-dart
```
Comment (osm-vishnukyatannawar) [inline coding-standards/flutter.md:7]: Is dart different from flutter?

File: coding-standards/flutter.md [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+run the following command in the terminal to fetch and install the newly added package:
+```
+flutter pub get
+```
+**Configure flutter_lints**:</br>
+Mac Android Studio: "Preferences/Settings" </br>
+Window Android Studio: go to "Languages & Frameworks" > "Dart" > "Lints". Click on the "Configure" link next to the "Package: flutter_lints" option.
+
+**Enable flutter_lints rules**:
+In the "Select Dart Lints" window, check the box next to "flutter_lints" to enable its lint rules. Click "OK" to apply the changes.
```
Comment (osm-vishnukyatannawar) [inline coding-standards/flutter.md:29]: flutter_lints - comes by default with some lint rules?

We don't need to add any additional configuration or rules on top it?

File: coding-standards/flutter.md [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+
+### **Dart Formatter:**
+**Configure and Enable dartfmt**:
+In the "Preferences/Settings" window, go to "Languages & Frameworks" > "Dart" > "Enable 'Run 'dartfmt' on save' for .dart files". This will automatically format your Dart code using dartfmt whenever you save a file.
```
Comment (osm-vishnukyatannawar) [inline coding-standards/flutter.md:33]: Like above you gave instructions for MAC and Windows. Do we need the same for this also?
Or it is common in both?

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]

File: coding-standards/flutter.md (hunk 1 above)
Comment (anand-OSM) [reply coding-standards/flutter.md:7]: Flutter is UI sdk comes with widgets and used for UI purposes, dart is client-side programming language use in flutter app development

File: coding-standards/flutter.md (hunk 2 above)
Comment (anand-OSM) [reply coding-standards/flutter.md:29]: Yes, flutter_lints comes with default recommended rules and as of now I don't think any addition on top of it is required.

File: coding-standards/flutter.md (hunk 3 above)
Comment (anand-OSM) [reply coding-standards/flutter.md:33]: In Mac its Preferences but in Window its Setting, else everything is common

File: coding-standards/flutter.md (hunk 2 above)
Comment (osm-vishnukyatannawar) [reply coding-standards/flutter.md:29]: Is there any link where people can go over that default configuration list and learn about it.
If yes can we add that.

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**